### DISCUSSION TITLE
Generic way to build first stage

### DISCUSSION THREAD

**Discussion by vonbox** [containers/buildah#5263] [Q&A]
With `buildah bud --target=layer0 .` I can build the first stage of a multistage containerfile. Is there a way to specify the first layer without me needing to name it, or without needing to know the name? Something like `buildah bud --target=0 .`?
I am trying to create a kinda generic script that should be able to build the first stage of any containerfile.

**Comment by nalind** [comment]
That's not something it currently supports, but it would be a fairly simple addition to https://github.com/openshift/imagebuilder/, which we use as a library inside of `buildah bud` to handle the `--target` flag.  I'll create an issue and transfer it there.

**Reply by nalind** [reply]
Opened https://github.com/openshift/imagebuilder/issues/274.
